fiftyone/server/mutation.py

```diff
@@ -200,14 +204,18 @@ async def set_view(
         form: t.Optional[StateForm] = None,
     ) -> t.Union[BSONArray, None]:
         state = get_state()
+        state.group_id = None
+        state.sample_id = None
         state.selected = []
         state.selected_labels = []
+
         if not dataset_name:
             state.dataset = None
-            state.view = None
+            state.group_slice
```

Comment:
Is this a type? Not assigned or used as condition

Reply:
yes, typo 😅 🙇